source/stylesheets/modules/_toc.scss

```diff
@@ -44,3 +44,7 @@
     display: none;
   }
 }
+
+.toc-link--in-view {
+  font-weight: bold;
```

Comment:
I had thought we were replicating the style of these pages - https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction

Do you think we need the background colour?

Regarding the bold active state – we can stop this by adding {letter-spacing: 0.03em} to the normal state and removing it on the active state. Essentially makes the text the same width as if it was bolded and doesn't look too dramatic. Thoughts?